google-cloud-bigquery/lib/google/cloud/bigquery/external/data_source.rb

```diff
@@ -37,7 +37,7 @@ module External
         #
         #   bigquery = Google::Cloud::Bigquery.new
         #
-        #   avro_url = "gs://bucket/path/to/data.avro"
+        #   avro_url = "gs://bucket/path/to/*.avro"
```

Comment:
Why are we changing this? Does the call support wildcards like this?

Reply:
Yes, see [Wildcard support for Cloud Storage URIs](https://cloud.google.com/bigquery/external-data-cloud-storage#wildcard-support). And Avro data will typically be in multiple files, see examples in [Loading Avro data from Cloud Storage](https://cloud.google.com/bigquery/docs/loading-data-cloud-storage-avro). I believe showing a single file is less realistic.